internal/wildcard/match.go

```diff
@@ -81,3 +81,58 @@ func deepMatchRune(str, pattern []rune, simple bool) bool {
 	}
 	return len(str) == 0 && len(pattern) == 0
 }
+
+// Index - finds a position of substring which matches/satisfies the pattern string.
+// Supports  '*' and '?' wildcards in the pattern string.
+// If nothing is found, it returns -1.
+func Index(pattern, name string) (index int) {
+	if pattern == "" || pattern == "*" {
+		return 0
+	}
+	rname := make([]rune, 0, len(name))
+	rpattern := make([]rune, 0, len(pattern))
+	for _, r := range name {
+		rname = append(rname, r)
+	}
+	for _, r := range pattern {
+		rpattern = append(rpattern, r)
+	}
+	return deepIndexRune(rname, rpattern, 0)
+}
+
+//deepIndexRune is recursive function which searches for a position of substring which matches given wildcard pattern
```

Comment:
Missing the space between / and deepIndexRune.
Also missing dot at the end of the sentence.
Also should write "is a recursive function"...

Reply:
fixed